src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -411,12 +432,24 @@ define('diagramStore', ['jquery', 'xwiki-meta', 'draw.io', 'xwiki-events-bridge'
     });
   };
 
+  var deleteFile = function(fileName) {
+    var deferred = $.Deferred();
+    var doc = new XWiki.Document(xm.documentReference);
+    var queryString = '?form_token=' + xm.form_token + '&amp;xredirect=' +
```

Comment:
I would add the ``?`` separator inside ``getAttachmentURL``. I would make the API easier to integrate with ``$.param()`` for instance, which doesn't output the ``?``. You would then be able to write:

```
var attachmentURL = xutils.getAttachmentURL(fileName, 'delattachment', $.param({
  'form_token': xm.form_token,
  'xredirect': doc.getURL('get', 'outputSyntax=plain')
}));
```

I would even go further and use ``$.param()`` inside ``getAttachmentURL`` if the query string is passed as an object and not as a string. Then you would be able to write:

```
var attachmentURL = xutils.getAttachmentURL(fileName, 'delattachment', {
  'form_token': xm.form_token,
  'xredirect': doc.getURL('get', 'outputSyntax=plain')
});
```